### PR TITLE
Improving benchmark experience

### PR DESCRIPTION
**Pull request by Lenoush**
- A lot of code modification 
- Add creation data function file
- Add creation 2D trajs file
- Add creation of yalm file ( for doing all the backend , all traj , all n_coils is the same time: can be do with hydra , i didn't know but the script is cool)
- Add graphs in Readme : waiting for comments

### CODE REVIEW

File: result3D_cpu.png
Comment (chaithyagr) [inline result3D_cpu.png:1]: We dont need pea GPU memory for all CPU versions

File: 0_create_data.py
Comment (paquiteau) [inline 0_create_data.py:1]: Name this `05_create_data.py` The number in front, tells in which order to run the scripts.

File: 50_auto_benchmark_perf.py
Comment (paquiteau) [inline 50_auto_benchmark_perf.py:1]: similarly remove the `50_`, this is a standalone file.

**Review by paquiteau — COMMENTED** [review]
Nice Work ! Figures looks very promising. 
Ideally we need more documentation in each file (a module docstring will do). Let's have in mind that this benchmark could be run by anyone.

File: 0_create_data.py
Comment (Lenoush) [reply 0_create_data.py:1]: we don't get the image before get the trajs ?

**Comment by chaithyagr** [comment]
I will let @paquiteau handle this PR's review.

**Review by paquiteau — CHANGES_REQUESTED** [review]
Congrats, the figures are really nice. 
In the README, write the exact commands you had to run to produce the results figures from start to finish, so that we can rerun the benchmark if any backend gets performance improvements

**Review by paquiteau — CHANGES_REQUESTED** [review]
We still need the exact commands you ran for getting those figures :)

**Pull request merged**